.github/workflows/ci.yml

```diff
@@ -99,15 +119,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: ['11', '17']
+        java: ['8', '11']
```

Comment:
The purpose of the task `testJvms` is to test on other JVM versions _besides the base one_, so it should stay `['8', '11']`.

Reply:
It should not, as the base one is 17.